x-pack/plugins/observability/public/hooks/slo/use_fetch_index_pattern_fields.ts

```diff
@@ -35,6 +35,9 @@ export function useFetchIndexPatternFields(
             query: {
               pattern: indexPattern,
             },
+            headers: {
+              'Elastic-Api-Version': 1,
```

Comment:
If the version changes, do we need to do something, or will the API ensure this version will also work?

Reply:
If they introduce a change and bump the version, they will have to keep the V1 anyway. So until we are not interested in getting the new thing, we can stay on the V1.